webrender/res/ps_radial_gradient.vs.glsl

```diff
@@ -23,12 +23,19 @@ void main(void) {
     // and not snap here?
     vStartCenter = floor(0.5 + gradient.start_end_center.xy * uDevicePixelRatio) / uDevicePixelRatio;
     vEndCenter = floor(0.5 + gradient.start_end_center.zw * uDevicePixelRatio) / uDevicePixelRatio;
-    vStartRadius = gradient.start_end_radius_extend_mode.x;
-    vEndRadius = gradient.start_end_radius_extend_mode.y;
+    vStartRadius = gradient.start_end_radius_ratio_xy_extend_mode.x;
```

Comment:
The start and end radius are given in terms of x and we're only scaling y so they're unaffected. I could rename the field to `start_end_radius_x_ratio_xy_extend_mode` or add a comment if that would make it more clear.